### ISSUE TITLE
Command Completion

### ISSUE THREAD

**Issue by DoggersFor1** [customrealms/core#5]
Command Completion

**Comment by BasToTheMax** [comment]
Some ide's have this automaticly.

**Comment by connerdouglass** [comment]
I think he means command completion in the Minecraft game itself, so players can see the list of options for commands and arguments. We will definitely need to implement this!

**Comment by DoggersFor1** [comment]
yes. There is currently a fix for this. go into the commands.yml file and add the commands. it the one generated with the project

**Comment by AL1L** [comment]
Not quite a fix @Xgod2205, but that is a workaround.

**Comment by connerdouglass** [comment]
I just made some improvements to the CLI project here: [customrealms/cli#2](https://github.com/customrealms/cli/pull/2)

You can now define your plugin's commands and permissions in your `package.json` file and it'll get included in your plugin.yml in your JAR file.

What else is typically done in a Bukkit plugin to get command completion? I've never done it personally.

**Comment by ghost** [comment]
Can there be a tutorial on how customrealms/cli#2 works? I cannot seem to get it to work.

**Comment by BasToTheMax** [comment]
CustomRealms is a bit dead ig
